backend/src/main/java/de/tomze/backend/controller/UserController.java

```diff
@@ -51,30 +53,46 @@ public UserController(UserService userService) {
         return notFound().build();
     }
 
-    @PostMapping()
+    @PostMapping("/api/tomze/register")
     public ResponseEntity<UserFromAppDto> createUser(@RequestBody UserFromAppDto userFromAppDto) {
-
         UserEntity createdUserEntity = userService.createUser(userFromAppDto);
         UserFromAppDto createdUserFromAppDto = mapUserFromAppDto(createdUserEntity);
         return ok(createdUserFromAppDto);
 
     }
 
-    @PutMapping("/user/{userName}")
-    public ResponseEntity<UserFromAppDto> updateUser(@PathVariable String userName, @RequestBody UserFromAppDto userFromAppDto){
+    @PutMapping("/api/tomze/user/{userName}")
+    public ResponseEntity<UserFromAppDto> updateUser(@AuthenticationPrincipal UserEntity authUser, @PathVariable String userName, @RequestBody UserFromAppDto userFromAppDto) {
```

Comment:
kannst dir überlegen nach jeder der 3 @ Methoden ein Return einzufügen das die jeweilig in einer neuen zeile sind. Find ich leichter zu lesen.